package-lock.json

```diff
@@ -1,8 +1,11147 @@
 {
     "name": "andculturecode-javascript-core",
     "version": "0.6.4",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
Looks like the massive change here was from updating `npm` to version 7 on my machine. V2 lockfiles should be backwards compatible with npm v5/6 so this shouldn't _really_ be an issue, but I can certainly revert it and regenerate it with an older version